core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -805,7 +806,7 @@ class DAGScheduler(
     }
 
     val properties = if (jobIdToActiveJob.contains(jobId)) {
-      jobIdToActiveJob(stage.jobId).properties
+      jobIdToActiveJob(stage.jobId).properties.orNull
```

Comment:
@pwendell , is there a good reason that SparkListenerStageSubmitted and SparkListenerJobStart (both \@DeveloperApi) are using 
```
properties: Properties = null
```
because otherwise, I would like to update these to Option as well.